File: manila/src/main/java/cz/fb/manaus/manila/filter/AbnormalPriceFilter.java
package cz.fb.manaus.manila.filter;

import com.google.common.collect.Range;
import cz.fb.manaus.reactor.price.AbstractPriceFilter;
import org.springframework.stereotype.Component;

@Component
public class AbnormalPriceFilter extends AbstractPriceFilter {

    public AbnormalPriceFilter() {
        super(3, 100, Range.all());
    }

}

File: manila/src/main/java/cz/fb/manaus/manila/filter/AbnormalPriceFilter.kt
package cz.fb.manaus.manila.filter

import cz.fb.manaus.reactor.price.PriceBulldozer
import cz.fb.manaus.reactor.price.PriceFilter
import org.springframework.context.annotation.Primary
import org.springframework.stereotype.Component

@Primary
@Component
class AbnormalPriceFilter(priceBulldozer: PriceBulldozer) :
        PriceFilter(3, 100.0, 0.0..100.0, priceBulldozer)
